README.md

```diff
@@ -15,6 +15,9 @@ Looking to see what we're up to or contribute? Check out one of our products:
 - [Focus for iOS](https://github.com/mozilla-mobile/focus-ios)
 
 ### Android
+*Want to know how to contribute on Android? Check out our
+[CONTRIBUTING](android/CONTRIBUTING.md) documentation! We encourage non-code
+contributions too!*
```

Comment:
Optional, provide examples of this, like QA, localization, accessibility testing, etc